rollup.config.mjs

```diff
@@ -81,6 +83,7 @@ function banner(bundleName, minified) {
       : 'banner'
   );
 
+  const __dirname = path.resolve();
```

Comment:
Was not aware that this is how it works. I always did [this](https://stackoverflow.com/a/50052194).

Reply:
If it works, great :)